woof-code/rootfs-skeleton/etc/init.d/rc.ucode

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+# service script to identify if ucode is present and load it if so
+# if not download it and load it if possible
+# GPLv2
+
+[ -e "$HOME/.ucode.rc" ] && . $HOME/.ucode.rc # written by micro2pup.sh
```

Comment:
is HOME set when running the init script? Why not /etc or ~?

Reply:
Fixed.